Core/Platform/Core.Platform.Shared/MobileCoreInitializer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using AeroGear.Mobile.Core.Exception;
+using Xamarin.Forms;
+
+namespace AeroGear.Mobile.Core.Platform
+{
+    public sealed class MobileCoreInitializer
```

Comment:
do we actually need this class? Why not just create new method in MobileCore to init with an `app`?

Reply:
DependecyService is not available outside of Xamarin project. Core is .NET Standard library. We can't have it there.

Reply:
actually, why do we need the `Core` project and the `Core.Platform.Shared` project? can we just have the `Core.Platform.Shared` project instead?